data-es6.js

```diff
@@ -16210,6 +16210,40 @@ exports.tests = [
       },
     },
     {
+      name: 'Number.parseFloat',
+      mdn: 'https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/parseFloat',
+      exec: function () {/*
+        return typeof Number.parseFloat === 'function';
```

Comment:
this should also test that `Number.parseFloat === global.parseFloat`, which the spec [requires](https://tc39.github.io/ecma262/#sec-number.parsefloat)